src/main/java/core/basesyntax/Calculational.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Calculational {
+    double getFigureArea();
```

Comment:
```suggestion
    double getArea();
```
let's not attach method name to Figure